### ISSUE TITLE
Add Adapter Trimming 

### ISSUE THREAD

**Issue by tobsecret** [nf-core/covid19#6]
Add the adapter trimming step from the galaxy pipeline which uses fastp. 
I will work on this!

**Comment by drpatelh** [comment]
Nice! Thanks @tobsecret! Thanks for updating the [Google sheet](https://docs.google.com/spreadsheets/d/1zDL-ugHWBexeK1x8zf5_tr_c0tLLhzgFZqc7l2toDNU/edit#gid=0) too 😎

**Comment by sb10** [comment]
I don't know if this would be the same or a different request, but what about doing primer trimming with something like ivar?

**Comment by tobsecret** [comment]
@sb10 That would be a different issue, as this issue is based on the original plan to port [this galaxy-based pipeline](https://github.com/galaxyproject/SARS-CoV-2) also not sure what other functionality ivar would offer that fastp doesn't. 

I could of course include the option to supply a file with adapters, but as is, fastp auto-detects and clips adapters, [see here](https://github.com/OpenGene/fastp#adapters] for an explanation. 

Currently, development of this pipeline is on hold until further notice, owing to the fact that it does not add much to the existing eco-system of pipelines. @drpatelh has [forked the Arctic pipeline to nf-core](https://github.com/nf-core/arctic), so any contributions should be made there.

**Comment by drpatelh** [comment]
Hi @sb10 ! Yes, as @tobsecret mentioned we have put the development of this pipeline on hold shortly after we created it. Not ideal but it made sense to have the domain knowledge from researchers before we pushed on.

Hopefully, we will be collaborating with the authors of the ARCTIC pipeline to implement `nf-core/arctic`. Its still early days with this pipeline too but may be worth joining the #artic channel on the [nf-core Slack workspace](https://nf-co.re/join) for updates and if you would like to contribute :)
